externals/react-native/v0/konfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "0.52.0-0",
+  "dependencies" : {
+    "subprojects" : [
```

Comment:
(minor) What is the idea behind the "subprojects" term? It sounds a bit confusing to me.

Why not just omit it altogether? Do we expect some other kinds of dependencies?

Reply:
Yes, the idea is that we can have some other kind of dependencies (npm and maven).
Other candidates that I've considered are "local" and "project".

Thoughts?

Reply:
I've decided just to use "dependencies" w/o nested map for this now.